src/components/account/CreateAccount.tsx

```diff
@@ -104,9 +105,9 @@ const CreateAccount = () => {
             (error) => error !== ''
           )
           if (validationErrors.length !== 0) {
-            setFormErrors(validationErrors)
-            setRequestStatus('validationerror')
+            setFormValidationErrors(validationErrors)
           } else {
+            setFormValidationErrors([''])
```

Comment:
This should be set to null right? An array with an empty string will appear truthy inside `RequestStatus` and prevent any messages related to the request from showing because it will be showing "validation errors".